packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -122,7 +123,14 @@ const AccountSwitcher = observer(({ history, is_mobile, is_visible }) => {
         closeAccountsDialog();
         if (account_loginid === loginid) return;
         await switchAccount(loginid);
-        Analytics.setAttributes({ account_type: loginid.substring(0, 2) });
+        Analytics.setAttributes({
+            account_type: LocalStore?.get('active_loginid')?.substring(0, 2) ?? 'unlogged',
+            app_id: getAppId(),
+            device_type: isDesktopOs() ? 'desktop' : 'mobile',
```

Comment:
Unsure about this function isDesktopOs()? It's pretty blunt... "windows" "mac" or "linux". Is this the standard way of assessing in our codebase?

Reply:
I found many places, include `client-store` where we using this setup. Since I'm not really familiar with codebase, I reused existing code